toolbox

```diff
@@ -711,6 +712,18 @@ create()
         monitor_host="--monitor-host"
     fi
 
+    echo "$base_toolbox_command: checking if 'podman create' supports --ulimit host" >&3
+
+    if man podman-create 2>&3 | grep "You can pass 'host'" >/dev/null 2>&3; then
```

Comment:
The single quotes around `host` broke the `grep` for me. Are they really needed?

Reply:
I removed the single quotes for the time being.